From 5755f6edbbc807bb6eb71b27eb810aab98157743 Mon Sep 17 00:00:00 2001
From: Hariprasad Kelam <hkelam@marvell.com>
Date: Fri, 16 Oct 2020 20:38:16 +0530
Subject: [PATCH 14/22] octeontx2-pf: add Support QSGMII mode

Update ethtool supported mode mask such that QSGMII port
will be visible in supported link modes.

Change-Id: If6a3f7da9b927201137dcb9ff26d94e09ba144ea
Signed-off-by: Hariprasad Kelam <hkelam@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/38067
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_ethtool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_ethtool.c b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_ethtool.c
index fdca6e85a97f..3081aada3f81 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_ethtool.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_ethtool.c
@@ -47,7 +47,7 @@ struct otx2_stat {
 	.index = offsetof(struct otx2_dev_stats, stat) / sizeof(u64), \
 }
 
-#define OTX2_ETHTOOL_SUPPORTED_MODES 0x638CFFB //110001110001100111111111011
+#define OTX2_ETHTOOL_SUPPORTED_MODES 0x638CFFF //110001110001100111111111111
 #define OTX2_ETHTOOL_ALL_MODES (ULLONG_MAX)
 
 static const struct otx2_stat otx2_dev_stats[] = {
-- 
2.17.1

